deps/wazuh_testing/wazuh_testing/qa_docs/doc_generator.py

```diff
@@ -55,12 +59,17 @@ def is_valid_file(self, file):
             - "file (str): File name to be controlled"
         returns: "boolean: False if the file should be ignored. True otherwise."
         """
+
         for regex in self.ignore_regex:
             if regex.match(file):
+                DocGenerator.LOGGER.warning(f"File validation: {regex} not matchin with {file}")
                 return False
+
         for regex in self.include_regex:
             if regex.match(file):
+                DocGenerator.LOGGER.warning(f"File validation: {regex} not matchin with {file}")
```

Comment:
```suggestion
                DocGenerator.LOGGER.warning(f"File validation: {regex} not matching with {file}")
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523